chat-widget/src/components/livechatwidget/livechatwidgetstateful/LiveChatWidgetStateful.tsx

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
Please remove this and fix the linting requirements, this makes me sad :(

Reply:
it shouldnt be part of the code shipped, my bad